setup.py

```diff
@@ -36,14 +36,10 @@
         'Programming Language :: Python :: 3.4',
     ],
     keywords='pyramid swagger validation',
-
     packages=find_packages(exclude=["contrib", "docs", "tests*"]),
-    package_data={
-        'pyramid_swagger': ['pyramid_swagger/swagger_spec_schemas/v1.2/*'],
-    },
```

Comment:
FYI, package_data should have been removed when we switched over to using swagger_spec_validator.